### PR TITLE
Tcp: rename fields and minor cleanups

### PR DESCRIPTION
**Pull request by dimitry-**
Some code cleanups, nothing special.

### CODE REVIEW

File: OrbitCore/Tcp.cpp [hunk 1]
```diff
@@ -9,95 +9,74 @@
...
-                       m_Socket.close();
+                       socket_.close();
                      }
                    }
 
   );
 }
 
-//-----------------------------------------------------------------------------
 void TcpConnection::ReadWebsocketMessage() {
```
Comment (dpallotti) [inline OrbitCore/Tcp.cpp]: We're keeping this commented code for now?

File: OrbitCore/Tcp.cpp [hunk 2]
```diff
@@ -9,95 +9,74 @@
...
   /*asio::async_read( m_Socket, asio::buffer( &m_WebSocketBuffer,
```
Comment (dpallotti) [inline OrbitCore/Tcp.cpp]: We're keeping this commented code for now?

Comment (dimitry-) [reply OrbitCore/Tcp.cpp]: I guess there is no point, removed it.

Comment (pierricgimmig) [reply OrbitCore/Tcp.cpp]: The websocket code used to work but was disabled when porting form Windows to Linux as it was no longer being used and I wanted to cut a dependency.  We can get rid of that code and revive it properly if we do need websockets again.

Comment (dpallotti) [reply OrbitCore/Tcp.cpp]: In that case we can also remove 
char m_WebSocketBuffer[MAX_WS_HEADER_LENGTH]

Comment (dimitry-) [reply OrbitCore/Tcp.cpp]: removed this.

**Pull request merged**